airflow/contrib/hooks/gcp_sheets_hook.py

```diff
@@ -0,0 +1,274 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+"""
+This module contains a Google Sheets API hook
+"""
+
+from googleapiclient.discovery import build
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+from airflow.exceptions import AirflowException
+
+
+class GCPSheetsHook(GoogleCloudBaseHook):
+    """
+    Interact with Google Sheets via GCP connection
+    Reading and writing cells in Google Sheet:
+    https://developers.google.com/sheets/api/guides/values
+    """
+
+    def __init__(self,
+                 gcp_conn_id: str = 'google_cloud_default',
+                 spreadsheet_id: str = None,
+                 api_version: str = 'v4',
+                 delegate_to: str = None) -> None:
+        """
+        :param grpc_conn_id: The connection ID to use when fetching connection info.
+        :type grpc_conn_id: str
+        :param gsheet_id: The Google Sheet ID to interact with
+        :type gsheet_id: str
+        :param range_: The Range or Named Range to interact with
```

Comment:
Didn't find the `range_` arg in the function signature.

Reply:
whoops that doesn't belong, I'll remove the comment, thanks for catching